release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -2030,6 +2030,14 @@ In the table below, features are marked with the following statuses:
 [id="ocp-4-4-known-issues"]
 == Known issues
 
+When upgrading to a new {product-title} z-stream release, connectivity to the API server might be interrupted as nodes are upgraded, causing API requests to fail.
++
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1791162[*BZ#1791162*])
```

Comment:
We only add the BZ link using a `+` for entries that have steps/bullets. This (and the one after) can be added to the end of the sentence.